strings/names/moth_female.txt

```diff
@@ -0,0 +1,16 @@
+Naenia
```

Comment:
cut this file

Reply:
why

Reply:
https://en.wikipedia.org/wiki/Naenia_(moth)

Reply:
moth names aren't gendered